src/core/conn.c

```diff
@@ -744,6 +749,10 @@ static int _conn_cfg_init_by_dsn(conn_cfg_t *cfg, char *ebuf, size_t elen)
   r = SQLGetPrivateProfileString((LPCSTR)cfg->dsn, "CONN_MODE", (LPCSTR)"0", (LPSTR)buf, sizeof(buf), "Odbc.ini");
   if (r == 1) cfg->conn_mode = !!atoi(buf);
 
+  buf[0] = '\0';
+  r = SQLGetPrivateProfileString((LPCSTR)cfg->dsn, "SCADA", (LPCSTR)"0", (LPSTR)buf, sizeof(buf), "Odbc.ini");
```

Comment:
On Windows, do you read DSN property values from the registry or the configuration file?

Reply:
registry